src/garage/tf/policies/categorical_cnn_policy.py

```diff
@@ -152,6 +153,7 @@ def get_action(self, observation):
             dict(numpy.ndarray): Distribution parameters.
 
         """
+        observation = convert_to_numpy(self.observation_space, observation)
```

Comment:
These really shouldn't be necessary for this policy, since if your input isn't a valid image, flattening it certainly won't make it into one.